packages/core/src/domain/session/oldCookiesMigration.ts

```diff
@@ -13,7 +13,8 @@ export const LOGS_SESSION_KEY = 'logs'
  * This migration should remain in the codebase as long as older versions are available/live
  * to allow older sdk versions to be upgraded to newer versions without compatibility issues.
  */
-export function tryOldCookiesMigration(sessionCookie: CookieCache) {
+export function tryOldCookiesMigration(options: CookieOptions) {
+  const sessionCookie = cacheCookieAccess(SESSION_COOKIE_NAME, options)
```

Comment:
Since `sessionCookie` will be read only once, does it need to be cached?

Reply:
No but we still need the logic of [persistSession](https://github.com/DataDog/browser-sdk/blob/main/packages/core/src/domain/sessionManagement.ts#L124) that take a cookieCache as input.
Since the plan is to remove cookie cache in the next PR, I was planning to deal with that refactoring then.

Reply:
Ok!